BREAKING.md

```diff
@@ -169,6 +158,29 @@ Note: `IConnectableRuntime` is only to be implemented internally, so removing th
 
 `IDocumentServiceFactory.protocolName` was deprecated in 2.0.0-internal.3.0.0 and has now been removed.
 
+### Changes to Summarizer's public API
```

Comment:
It should be mentioned somewhere that you can no longer request summarizer via the request pattern right?